Packages/com.chisel.editor/Chisel/Editor/Editor/MaterialBrowser/ChiselEmbeddedTextures.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using UnityEditor;
+using UnityEngine;
+
+namespace Chisel.Editors
+{
+    // color space adjusted texture utilities
+    internal static class ChiselEmbeddedTextures
```

Comment:
It might make sense to merge this with the ChiselMaterialManager, arguably ChiselEmbeddedTextures is a better name (although it also contains PhysicMaterial, so ChiselEmbeddedAssets maybe?)

Reply:
I think going with `ChiselEmbeddedAssets` would work the best, yeah. I'll look into this in a separate PR as I'm trying to avoid too many changes to other chisel components in this PR.

Reply:
Can you do this before I can approve this PR?

Reply:
Yeah, shouldn't be a problem.